src/viam/ur/module/ur_arm.hpp

```diff
@@ -173,4 +173,5 @@ class URArm final : public Arm, public Reconfigurable {
     std::shared_mutex config_mutex_;
     std::unique_ptr<state_> current_state_;
     std::optional<ports> current_ports_;
+    std::optional<double> reject_move_request_threshold_deg;
```

Comment:
- The `reject_move_request_threshold_deg` field should live on `struct state_` with the other configuration parameters. The `current_ports_` should be there too, but I've already cleaned that up in my tree.
- I think it might end up being simpler to store it in radians after validating it.